feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-image/configurator-attribute-single-selection-image.component.ts

```diff
@@ -77,4 +79,8 @@ export class ConfiguratorAttributeSingleSelectionImageComponent
       isLightedUp: value ? value.selected : false,
     };
   }
+
+  getAriaLabel(attribute: Configurator.Attribute, value: Configurator.Value) {
```

Comment:
There is no need to define this method. 
`getAriaLabelGeneric` could be called directly in HTML tree.
See `createAttributeUiKey` in `configurator-attribute-header.component.html`